src/lib/stitching/vortex/__tests__/rankedStats.test.ts

```diff
@@ -33,11 +33,7 @@ describe("AnalyticsRankedStats type", () => {
     query($id: String!) {
       partner(id: $id) {
         analytics {
-          topArtworks: rankedStats(
-            first: 1
-            objectType: ARTWORK
-            period: ONE_YEAR
-          ) {
+          rankedStats(first: 1, objectType: ARTWORK, period: ONE_YEAR) {
```

Comment:
**Situation 2**
something seems broken with aliases inside tests - I spent a few weeks working on it and didn't figure out the root cause of it. It might be something small but I can't figure it out. Also I tried this in production and the query is working well.

<img width="1503" alt="Screenshot 2024-07-23 at 17 39 54" src="https://github.com/user-attachments/assets/8b6c3eae-2d2c-4d8e-8ad9-f1c615a9a2e5">

🔙 [Go back](https://github.com/artsy/metaphysics/pull/5838/commits/38f51fdd10c12492d4000473e49317f7cb499a38#r1689374081)